### PR TITLE
docs: improve Android custom code section

### PR DESCRIPTION
**Pull request by RoderickQiu**
- Android custom code guide modified. 
> I was stuck on that the other day and I've solved it. And today I saw this article again and decide to modify it.
I think it's now better for common users to comprehend.
- Some style changes.

### CODE REVIEW

File: site/docs-md/guides/community.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ contributors:
 
 [Capacitor Workflow for iOS and Android Apps &#8250;](https://www.youtube.com/watch?v=oXbRcpsytGQ)
 
-[Using the Capacitor in Vue.js app](https://akhromieiev.com/tutorials/using-capacitor-in-vue-js-app/)
+[Using the Capacitor in Vue.js app &#8250;](https://akhromieiev.com/tutorials/using-capacitor-in-vue-js-app/)
 
 ## APIs
 
-[Camera: Building an Ionic Framework Camera App](/docs/guides/ionic-framework-app)
+[Camera: Building an Ionic Framework Camera App &#8250;](/docs/guides/ionic-framework-app)
```
Comment (jcesarmobile) [inline site/docs-md/guides/community.md]: I think the `&#8250;` is only intended for external links
```suggestion
[Camera: Building an Ionic Framework Camera App](/docs/guides/ionic-framework-app)
```

File: site/docs-md/guides/community.md [hunk 2]
```diff
@@ -18,23 +18,23 @@ contributors:
...
 
 [Network: Ionic 4 and Network Detection with Capacitor &#8250;](https://developer.school/posts/ionic-4-network-detection-with-capacitor/)
 
-[Push Notifications: Using Push Notifications with Firebase in an Ionic Angular App](/docs/guides/push-notifications-firebase)
+[Push Notifications: Using Push Notifications with Firebase in an Ionic Angular App &#8250;](/docs/guides/push-notifications-firebase)
```
Comment (jcesarmobile) [inline site/docs-md/guides/community.md]: I think the `&#8250;` is only intended for external links
```suggestion
[Push Notifications: Using Push Notifications with Firebase in an Ionic Angular App](/docs/guides/push-notifications-firebase)
```

File: site/docs-md/guides/community.md (hunk 1 above)
Comment (RoderickQiu) [reply site/docs-md/guides/community.md]: OK

**Comment by RoderickQiu** [comment]
Finished then :D

**Pull request merged**